Source/scrollrt.cpp

```diff
@@ -1061,7 +1062,7 @@ static void DrawFPS()
 			frameend = 0;
 		}
 		wsprintf(String, "%d FPS", framerate);
-		PrintGameStr(8, 65, String, COL_RED);
+		PrintGameStr(8+8, 65, String, COL_RED);

```

Comment:
Could you elaborate what is truncating this part of the screen?

Reply:
On my tv/monitor (LG), the left and right 8 pixels of any 640x480 screens is not displayed. This was a tmp fix for my setup. I shouldn't have make this leaked out but I don't know if it is possible to mark some specific changed lines as "invisible" for git.

Reply:
Ok that's what I expected :) I normally commit lines/block individually which helps catch stuff like this. But I also realize that you didn't make this PR so your might not have felt that it was totally ready, and you didn't really have a chance to write a summery either, but you have been very helpful in commenting on the changes here in the review so thank you for that :)